XamlStyler3.Package/XamlStyler3.Package.csproj

```diff
@@ -56,7 +56,22 @@
     <DeployExtension>false</DeployExtension>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="envdte, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
+      <EmbedInteropTypes>True</EmbedInteropTypes>
```

Comment:
I believe this should be set to false. It resolves a conflict with reference to Microsoft.VisualStudio.Shell.15.0